README.md

```diff
@@ -47,7 +47,7 @@ it, simply add the following line to your Podfile:
 
 ```ruby
 use_frameworks!
-pod "CollectionViewSlantedLayout", '~> 3.0'
+ pod 'CollectionViewSlantedLayout', '~> 3.0.1'
```

Comment:
Thx @WayneEld for your contribution. 

The *PATCH*  shouldn't be part of the version. Could you change it to
```ruby 
pod 'CollectionViewSlantedLayout', '~> 3.0'
```
Then i ll merge it with pleasure